src/game/client/shadow/w3dprojectedshadow.cpp

```diff
@@ -994,8 +1002,14 @@ int W3DProjectedShadowManager::Render_Shadows(RenderInfoClass &rinfo)
                         }
 
                         projector->Peek_Material_Pass()->UnInstall_Materials();
+
+                        Coord3D sphere_pos;
+                        sphere_pos.x = sphere.Center.X;
+                        sphere_pos.y = sphere.Center.Y;
+                        sphere_pos.z = sphere.Center.Z;
+
                         SimpleObjectIterator *iter = g_thePartitionManager->Iterate_Objects_In_Range(
```

Comment:
Have you checked to confirm it doesn't modify sphere? I'm sure it doesn't but best to check

Reply:
`PartitionManager::Iterate_Objects_In_Range` is not yet implemented. But the `Coord3D` is given as const, so it should not modify it, unless someone uses `const_cast` in there.

Reply:
Good enough for me